health/service/src/main/java/com/cerner/beadledom/health/HealthDependency.java

```diff
@@ -9,7 +9,7 @@
  */
 public abstract class HealthDependency {
 
-  private Boolean primary = false;
+  private Boolean primary = true;
```

Comment:
This is a change from defaulting the HealthDependencies from secondary to primary. Because we don't know if our consumers rely on the previous behavior (of `meta/health` checking dependencies if they were uncategorized) one of the solutions we came up with was to make the dependencies default to primary if unspecified. This change preserves the functionality of `meta/health` for projects that don't specify dependency  importance but I'm not sure that it is the correct way to handle this. Feedback would be appreciated.

Reply:
This is basically passive.

I think the only change in behavior will be for any dependency implementations that explicitly set `primary` to false, they will actually be excluded from the health check. Arguably this was their intent anyway.

Reply:
Would it be beneficial to add an ADR directory and include a small ADR documenting our decision on this?

Reply:
Added here: https://github.com/nab0310/beadledom/commit/6a0bd44aac6b9099379a8be557aeabc47089dafa